kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/AdaptiveBigQueryWriter.java

```diff
@@ -117,10 +134,15 @@ private boolean isTableMissingSchema(BigQueryException exception) {
         return writeResponse.getInsertErrors();
       }
       attemptCount++;
-      if (attemptCount >= AFTER_UPDATE_RETY_LIMIT) {
+      if (attemptCount >= AFTER_UPDATE_RETRY_LIMIT) {
```

Comment:
I feel so because the integration tests all pass each time, and also according to the experiment here, it takes less than 2.5 min for the async process: https://github.com/googleapis/google-cloud-go/issues/975

Reply:
Can we make AFTER_UPDATE_RETRY_LIMIT configurable?

Reply:
(We don't necessary do it now)